octodns/provider/ns1.py

```diff
@@ -146,25 +175,46 @@ def populate(self, zone, target=False, lenient=False):
         try:
             nsone_zone = self._client.loadZone(zone.name[:-1])
             records = nsone_zone.data['records']
+            geo_records = nsone_zone.search(has_geo=True)
```

Comment:
Would things with `has_geo=True` not show up in `.data['records']` or is this relying on the order of processing below to replace the plain record with the geo version?

Reply:
things with has_geo=True do show up in .data["records"], but not with their geo information.  Doing the `nsone_zone.search(has_geo=True)` gets us the records, along with their geo information.

Reply:
Looks like `.search` was added recently so `setup.cfg` will likely need to update the required version of `nsone`. I was getting test failures until I bumped it.